app/bundles/EmailBundle/Config/config.php

```diff
@@ -59,6 +59,11 @@
                 'path'       => '/email/view/{idHash}',
                 'controller' => 'Mautic\EmailBundle\Controller\PublicController::indexAction',
             ],
+            'mautic_email_oneclick_unsubscribe' => [
+                'path'       => '/email/unsubscribe/{idHash}',
+                'controller' => 'Mautic\EmailBundle\Controller\PublicController::oneClickUnsubscribeAction',
+                'method'     => 'POST',
+            ],
```

Comment:
Is the only difference between this new route and the one bellow (which is the same route) only the HTTP method? Can you add the GET method to the next one so it's explicitly visible what the difference is? I'm wondering whether it shouldn't be a different route, but it's just a feeling that it could cause issues, can't come up with a solid reason.

Reply:
That's why the test has failed - the current unsubscribe route seems to be using the POST for saving the preferences. I'll move the implementation to the current route. We should use the same URL to keep compatibility with RFC 2369 and the new RFC 8058 - there must be only one https URL in the List-Unsubscribe header.